### PR TITLE
2.03 add location to results baseline

### LINKED ISSUE #365: Add new location element (as a child of the baseline element)
Add a new element `iati-activities/iati-activity/result/indicator/baseline/location` with definition:

> The sub-national geographical identification of the target locations already defined and described in the iati-activity/location element. These can be described by gazetteer reference, coordinates, administrative areas or a textual description. Any number of locations may be reported.

The element will have an occurrence rule of `0..*` and will replicate the existing `iati-activities/iati-activity/result/indicator/period/actual/location` element.

This will also add the following sub-attributes with definitions:

- `iati-activities/iati-activity/result/indicator/baseline/location/@ref` - occurs `0..1`, type `xsd:string`
> A cross-reference to the internal reference assigned to a defined location: iati-activity/location/@ref.

**Tasks:**

- [x] Add tests
- [x] Add `iati-activities/iati-activity/result/indicator/baseline/location`
- [x] Add `iati-activities/iati-activity/result/indicator/baseline/location/@ref`

### PR DESCRIPTION
**Pull request by allthatilk**
Fixes #365 

Merge after #377

### CODE REVIEW

**Comment by dalepotter** [comment]
Suggest review awaits merging of #377.

**Comment by hayfield** [comment]
#377 now merged

File: tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/location/element-exists.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <result type="xx">
+      <title>
+        <narrative>Sample text</narrative>
+      </title>
+      <indicator measure="xx">
+        <title>
+          <narrative>Sample text</narrative>
+        </title>
+        <baseline year="2017" value= "xx">
+          <location></location><!-- optional location element included -->
```
Comment (hayfield) [inline tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/location/element-exists.xml]: Suggest noting in the comment that an optional attribute is missing.

File: tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/location/element-exists.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+
+<iati-activities version="xx">
+  <iati-activity> <!-- at least 1 -->
```
Comment (hayfield) [inline tests/should-pass/iati-activities/iati-activity/result/indicator/baseline/location/element-exists.xml]: Misc comments to remove

**Review by hayfield — CHANGES_REQUESTED** [review]

**Comment by hayfield** [comment]
Flipping label since I forgot to do that with the review.

**Review by hayfield — APPROVED** [review]

**Pull request merged**